frontend/src/bundles/common/components/chat-list/chat-list.tsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useEffect, useState } from '../../hooks/hooks.js';
+import { Grid } from '../components.js';
+import { ChatListItem, ChatListSearch } from './components.js';
+import styles from './styles.module.scss';
+
+const EMPTY_ARRAY = 0;
```

Comment:
Raname and move it to constants folder 

```suggestion
const EMPTY_ARRAY_LENGTH = 0;
```